pages/home.js

```diff
@@ -17,21 +18,26 @@ const download = (store, actions) => {
     fetch('/puller/status').then((resp) => resp.json())
     .then((resp) => {
       actions.setData(resp);
+      actions.setTime();
     });
     
   }
 };
 
+
 module.exports = (store, actions) => {
-  download(store, actions);
+  window.setInterval(() => {
+    download(store, actions);
+  }, 5000);
```

Comment:
I would potentially have this logic in an action that triggers the fetch.
You need some kind of check here to make sure you only do this once (which is why having a dedicated reducer is useful).

Reply:
see: https://github.com/JRJurman/binary-clock/tree/master/actions